subprojects/packagefiles/gmp/meson.build

```diff
@@ -0,0 +1,548 @@
+project(
+  'gmp',
+  'c',
+  version : '6.2.1',
+)
+
+add_languages('cpp', required: get_option('cxx'))
```

Comment:
`cxx` was the original name in the configure script, I'm tempted to rename it to cpp, opinions on this ?